### PR TITLE
Turns Claw into a menace

### PR DESCRIPTION
**Pull request by EgorDinamit**
## About The Pull Request

- Claw will now use targetted serum W, taking away random agent(or clerk) if there's too many people in the area to a random department center.
- Claw's "serum A"(rename pending) will now completely ignore the "target", instead damaging everything around it in a very big AOE. Don't move around or you'll end up killing everyone around you!
- Added wide slash attack, which deals white damage in a somewhat big area.

## Why It's Good For The Game

- Allows Claw to ignore your petty attempts at murderballing it. 5 v 1? How about 1 v 1?
- It was always the main idea of it - don't move and everything will be fine. Now it's even more apparent.
- Claw was missing any sources of white damage, and this may still be not enough, but we shall see.

### CODE REVIEW

**Comment by Tonklaatech** [comment]
Why not just revert to pre 20 nerfs claw seems like a easy change

**Comment by Kitsunemitsu** [comment]
Is this really necessary?

**Comment by EgorDinamit** [comment]
Yes, it is.
It's both a nerf in some aspects and buff in the others. Outside of that it just makes things cooler(target W serum for example).

File: ModularTegustation/tegu_mobs/the_claw.dm [hunk 1]
```diff
@@ -216,9 +254,79 @@
...
+	to_chat(L, "<span class='danger'>The [src] is going to hunt you down!</span>")
+	addtimer(CALLBACK(src, .proc/TargetEviscerate, target, uhoh), 15)
+
+/mob/living/simple_animal/hostile/megafauna/claw/proc/TargetEviscerate(mob/living/L, obj/effect/eff)
+	if(!istype(L))
+		return FALSE
+	new /obj/effect/temp_visual/emp/pulse(src.loc)
+	icon_state = icon_living
+	visible_message("<span class='warning'>[src] blinks away!</span>")
+	var/turf/tp_loc = get_step(L, pick(1,2,4,5,6,8,9,10))
```
Comment (dsmith328) [inline ModularTegustation/tegu_mobs/the_claw.dm:278]:
```suggestion
	var/turf/tp_loc = get_step(L, pick(GLOB.alldirs))
```
Doesn't change functionality but it's a lot more readable.

File: ModularTegustation/tegu_mobs/the_claw.dm [hunk 2]
```diff
@@ -216,9 +254,79 @@
...
+		forceMove(target_turf)
+		for(var/mob/living/LL in range(1, target_turf)) // Attacks everyone around.
+			if(faction_check_mob(LL))
+				continue
+			if(LL == L)
+				continue
+			to_chat(LL, "<span class='userdanger'>\The [src] slashes you!</span>")
+			LL.apply_damage(15, BLACK_DAMAGE, null, LL.run_armor_check(null, BLACK_DAMAGE))
+			new /obj/effect/temp_visual/cleave(get_turf(LL))
+		tp_loc = get_step(src, pick(1,2,4,5,6,8,9,10))
```
Comment (dsmith328) [inline ModularTegustation/tegu_mobs/the_claw.dm:314]:
```suggestion
		tp_loc = get_step(src, pick(GLOB.alldirs))
```

**Review by dsmith328 — COMMENTED** [review]
Gotta see it in action at this point

**Review by LanceSmites328 — APPROVED** [review]

**Pull request merged**